lib/node/http.js

```diff
@@ -125,6 +125,7 @@ class HTTP extends Server {
           height: this.chain.height,
           tip: this.chain.tip.hash.toString('hex'),
           treeRoot: this.chain.tip.treeRoot.toString('hex'),
+          txn: this.chain.db.txn.rootHash().toString('hex'),
```

Comment:
What is a better name for this when it is exposed via the API? It is called `txn` internally, like a database transaction, but externally that may not make the most sense because it could be confused with an exchange of value transaction.